### PR TITLE
CANS-360 Fix 

### PR DESCRIPTION
**Pull request by vladimirPe**
## Description
Fix of "Getting blank page instead of error message when acc…essing assessment of sealed and sensitive children"
## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Layout/BreadCrumbsBuilder.test.js [hunk 1]
```diff
@@ -72,4 +72,12 @@ describe('bread crumb builder', () => {
     assertChildYouthListCrumbIsPresent(component);
     assertChildInfoCrumbIsPresent(component);
   });
+
+  describe('when navigate to Assessment and no client info', () => {
```
Comment (edgibbs) [inline app/javascript/Application/components/Layout/BreadCrumbsBuilder.test.js]: Just curious, but how would they get there without a client?

Comment (LeonidMarushevskyi) [reply app/javascript/Application/components/Layout/BreadCrumbsBuilder.test.js]: @edgibbs We assume the possibility of direct link

Comment (edgibbs) [reply app/javascript/Application/components/Layout/BreadCrumbsBuilder.test.js]: Alright.

**Comment by edgibbs** [comment]
retest this please

**Pull request merged**